src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -871,46 +877,92 @@ void makeNotNullMembersMaybeNull()
                     }
                 }
 
+                return;
+
                 ImmutableArray<Symbol> getMembersNeedingDefaultInitialState()
                 {
                     if (_hasInitialState)
                     {
                         return ImmutableArray<Symbol>.Empty;
                     }
 
+                    bool includeRequiredMembers = true;
+                    bool includeBaseRequiredMembers = true;
+                    bool hasThisConstructorInitializer = false;
+
+                    if (method is SourceMemberMethodSymbol { SyntaxNode: ConstructorDeclarationSyntax { Initializer: { RawKind: var initializerKind } } })
```

Comment:
Do we need special handling for record scenario (doesn't use `base` but rather a base type name): `record Derived : Base(...) ...`?

Reply:
You can't apply attributes to record primary constructors.